src/Cardano/Wallet/Api.hs

```diff
@@ -0,0 +1,28 @@
+{-# LANGUAGE DataKinds #-}
+{-# LANGUAGE TypeOperators #-}
+
+module Cardano.Wallet.Api where
+
+import Data.Proxy
+    ( Proxy (..) )
+import Servant.API
+    ( (:<|>), (:>), Capture, Delete, Get, JSON )
+
+import qualified Cardano.Wallet.Api.Types.Wallet as T
+import qualified Cardano.Wallet.Api.Types.WalletId as T
+
+type Api = DeleteWallet :<|> GetWallet :<|> ListWallets
+
+api :: Proxy Api
+api = Proxy
+
+type DeleteWallet = "wallets"
+    :> Capture "walletId" T.WalletId
+    :> Delete '[] ()
```

Comment:
I think we still want the content type to get the headers set correctly, and use [NoContent](https://hackage.haskell.org/package/servant-0.16/docs/Servant-API-ContentTypes.html#t:NoContent) from servant for the response type.

Reply:
Done.